migrations/20260615194915_position_view_reorged_marker.sql

```
@@ -0,0 +1,17 @@
+-- `$.Live.last_reorged_at` is non-null -- an append-only marker, since the
+-- original fill events and their reversal both stay in history.
+--
+-- SQLite only allows VIRTUAL (not STORED) generated columns via ALTER TABLE
+-- ADD COLUMN; VIRTUAL is fine here and still indexable.
+
+ALTER TABLE position_view
+    ADD COLUMN last_reorged_at TEXT
+    GENERATED ALWAYS AS (json_extract(payload, '$.Live.last_reorged_at')) VIRTUAL;
+
+CREATE INDEX IF NOT EXISTS idx_position_view_last_reorged_at
+    ON position_view(last_reorged_at) WHERE last_reorged_at IS NOT NULL;
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)